utilities/mobile_server_pool.py

```diff
@@ -210,8 +278,30 @@ def is_jenkins_job_running(job_name='Net-windows-TestSever-Funtional-tests', bui
             pool_json_str = '{ "ips": ['
             for node in node_list:
                 pool_json_str += '"{}", '.format(node)
+            if opts.sgw_nodes:
+                sgw_node_list = opts.sgw_nodes.strip().split(',')
+                for node in sgw_node_list:
+                    pool_json_str += '"{}", '.format(node)
+            if opts.load_balancer_nodes:
+                lp_node_list = opts.load_balancer_nodes.strip().split(',')
+                for node in lp_node_list:
+                    pool_json_str += '"{}", '.format(node)
+            pool_json_str = pool_json_str.rstrip(', ')
+            pool_json_str += "],"
+            if opts.sgw_nodes:
```

Comment:
I feel,  if we can keep only One if condition for opts.sgw_nodes:  and keep all the related code , so it will be very easy to read. (this way we might need to add two duplicate line )

Reply:
yeah I already tried to do that. Closing quote will be different if u have one if condition . json is not getting created properly